packages/primitives/src/components/Combobox/Combobox.tsx

```diff
@@ -1132,7 +1132,7 @@ const ComboboxItemImpl = React.forwardRef<ComboboxItemImplElement, ItemImplProps
       {...restProps}
       id={id}
       ref={composedRefs}
-      onPointerUp={composeEventHandlers(restProps.onPointerUp, handleSelect)}
+      onClick={composeEventHandlers(restProps.onClick, handleSelect)}
```

Comment:
Why change from onPointerUp to onClick?

Reply:
because of a strange behaviour with keyboard navigation, when I clicked the button it triggered the handler three times, one for pointer up, one for pointer down and one for click

Reply:
btw I reverted back the onPointerUp